.travis.yml

```diff
@@ -35,6 +35,25 @@ deploy:
     on:
       tags: true
 
+  # Create the npm tarball, extract, and deploy files to Artifactory for development builds
+  - provider: script
+    script:
+      - source travis/createExtractTarball.sh
```

Comment:
Ditch the source, favor ./ instead

Reply:
The file name describes the steps executed in the script, but it is better to name it based on its semantic purpose, like prepareNpmPackage or something like that.

Reply:
What’s the name “develop” doing here? That’s a branch name; we’re moving away from branch names in published assets.

Reply:
I was using `source` to export the `NPM_TMP_DIR` environment variable, which the regular invocation doesn't do if I remember correctly. Unfortunately, the travis environment doesn't support `source` so I'm currently searching for an alternative.

I do like `prepareNpmPackage` although, I don't want to suggest this command is required for any npm publication actions.

`develop` was a placeholder for the web equivalent of `SNAPSHOT`

Reply:
source makes sense for the reason you gave, but I gather that you’ll just drop it per your comment below on the temp for.

Reply:
We can test publishing to SNAPSHOT now

Reply:
What about "unstable" as an alternative to SNAPSHOT?

SNAPSHOT is exactly what we are aiming for in Maven terms, but I haven't seen its use in any of the javascript libraries. I think it is confusing when the context's are mixed and unfamiliar to the broad javascript community. I'm also worried about confusion from parties with maven projects finding the SNAPSHOT version qualifier. Selecting a naming pattern that is more web-centric (lower case and not mixing any Maven or other unassociated build systems) will provide a more natural use case for web developers.

Reply:
A web centric name sounds best, but we’re certainly not using unstable.

If there’s no clear choice, let’s drop it for this sprint.